doc/api/timers.md

```diff
@@ -123,6 +123,14 @@ Calling `timeout.unref()` creates an internal timer that will wake the Node.js
 event loop. Creating too many of these can adversely impact performance
 of the Node.js application.
 
+### timeout[Symbol.toPrimitive]()
```

Comment:
This is rendered as an empty link [in the GitHub](https://github.com/apapirovski/node/blob/d72b7ba0f1e44a3c6cd5a505b92b9397663a8986/doc/api/timers.md#timeoutsymboltoprimitive) and in HTML:

![l](https://user-images.githubusercontent.com/10393198/41067054-d7c15df0-69ec-11e8-80cb-415c5a0438aa.png)

Reply:
How to fix:

https://github.com/nodejs/node/blame/7588ceaf353af0f257d4d832bace4600edac704e/doc/api/stream.md#L1214

Reply:
\+ Should this section be placed before the `timeout.unref()` heading ABC-wise?

Reply:
I guess it depends on what convention we follow for symbols. I prefer it after but it could also be the first thing in the list. I wouldn't put it between ref & unref.

Will fix the link issue 👍